src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Validators/PreviewTokenCreateRequestTests.cs

```diff
@@ -0,0 +1,201 @@
+            Label = "Test",
+            Activates = FixedUtcNow.AddDays(6),
+            Expires = FixedUtcNow.AddDays(7)
+        };
+
+        var result = validator.TestValidate(request);
+        result.ShouldNotHaveValidationErrorFor(r => r.Activates);
+    }
+
+    [Fact]
+    public void Activates_After7Days_Fails()
+    {
+        var validator = new PreviewTokenCreateRequest.Validator(GetTimeProvider());
+        var request = new PreviewTokenCreateRequest
+        {
+            DataSetVersionId = Guid.NewGuid(),
+            Label = "Test",
+            Activates = FixedUtcNow.AddDays(7).AddSeconds(1),
+            Expires = FixedUtcNow.AddDays(9)
+        };
+
+        var result = validator.TestValidate(request);
+        result.ShouldHaveValidationErrorFor(r => r.Activates)
+            .WithErrorMessage("Activates date must be within the next 7 days.");
+    }
+
+    [Fact]
+    public void Expires_BetweenActivatesAndActivatesPlus7Days_Passes()
+    {
+        var validator = new PreviewTokenCreateRequest.Validator(GetTimeProvider());
+        var request = new PreviewTokenCreateRequest
+        {
+            DataSetVersionId = Guid.NewGuid(),
+            Label = "Test",
+            Activates = FixedUtcNow.AddDays(1),
+            Expires = FixedUtcNow.AddDays(5)
+        };
+
+        var result = validator.TestValidate(request);
+        result.ShouldNotHaveValidationErrorFor(r => r.Expires);
+    }
+
+    [Fact]
+    public void Expires_BeforeActivates_Fails()
+    {
+        var validator = new PreviewTokenCreateRequest.Validator(GetTimeProvider());
+        var activates = FixedUtcNow.AddDays(2);
+        var request = new PreviewTokenCreateRequest
+        {
+            DataSetVersionId = Guid.NewGuid(),
+            Label = "Test",
+            Activates = activates,
+            Expires = activates.AddSeconds(-1)
+        };
+
+        var result = validator.TestValidate(request);
+        result.ShouldHaveValidationErrorFor(r => r)
+            .WithErrorMessage("Activates date must be before the expires date.");
+    }
+
+    [Theory]
+    [InlineData(0)]
+    [InlineData(1)]
+    public void Expires_MoreThan7DaysAfterActivates_FailsIfOverBySecond(int second)
+    {
+        var validator = new PreviewTokenCreateRequest.Validator(GetTimeProvider());
+        var testDate = FixedUtcNow;
+        var request = new PreviewTokenCreateRequest
+        {
+            DataSetVersionId = Guid.NewGuid(),
+            Label = "Test",
+            Activates = testDate,
+            Expires = testDate.AddDays(7).AddSeconds(second)
+        };
+
+        var result = validator.TestValidate(request);
+        if (second > 0)
+        {
+            result.ShouldHaveValidationErrorFor(r => r)
+                .WithErrorMessage("Expires date must be no more than 7 days after the activates date.");
+        }
+        else
+        {
+            result.ShouldNotHaveValidationErrorFor(r => r.Expires);
+        }
+    }
```

Comment:
I don't think you really need to test this with theory data here and have logic within the test, because the case where expires is not more than 7 days after activates should already be covered by another success test.

So I would remove the `int second` parameter here and `.AddDays(7).AddSeconds(1)` instead.